prisma/schema.prisma

```diff
@@ -42,14 +42,16 @@ model sessions {
 }
 
 model users {
-  id             Int       @id @default(autoincrement())
-  name           String?
-  email          String?   @unique
-  email_verified DateTime?
-  image          String?
-  created_at     DateTime  @default(now())
-  updated_at     DateTime  @default(now())
-  players        players[]
+  id                                                       Int                   @id @default(autoincrement())
+  name                                                     String?
+  email                                                    String?               @unique
+  email_verified                                           DateTime?
+  image                                                    String?
+  created_at                                               DateTime              @default(now())
+  updated_at                                               DateTime              @default(now())
+  players                                                  players[]
+  viewing_permissions_usersToviewing_permissions_viewee_id viewing_permissions[] @relation("usersToviewing_permissions_viewee_id")
+  viewing_permissions_usersToviewing_permissions_viewer_id viewing_permissions[] @relation("usersToviewing_permissions_viewer_id")
```

Comment:
It seems that the introspection result/generated schema isn't capturing the greatest of names. Let's maybe try:
```suggestion
  viewedByPermissions viewing_permissions[] @relation("UsersToViewees")
  viewerPermissions viewing_permissions[] @relation("UsersToViewers")
```